### PR TITLE
[CAP-212] Create list of products

### PR DESCRIPTION
**Pull request by lpelizzaMS**
### Background
Create the screen to show a list of products

### Screenshot
rn-listproduct

### CODE REVIEW

File: ReactNative/groceryList/src/components/ProductItem.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    );
+  }
+}
+
+const styles = {
+  textStyle: {
+    fontSize: 18,
+    margin: 15
+  }
+};
```
Comment (mravinale) [inline ReactNative/groceryList/src/components/ProductItem.js:33]: Let's try to import styles

File: ReactNative/groceryList/src/actions/ProductActions.js [hunk 2]
```diff
@@ -23,4 +24,16 @@ export const productCreate = ({ name, count, brand }) => {
...
+export const productsFetch = () => {
+	const { currentUser } = firebase.auth();
+
+	return (dispatch) => {
+		firebase.database().ref(`/users/${currentUser.uid}/products`)
+			.on('value', snapshot => {
+				dispatch({ type: PRODUCTS_FETCH_SUCCESS, payload: snapshot.val() });
+
+		});
+	};
```
Comment (mravinale) [inline ReactNative/groceryList/src/actions/ProductActions.js:39]: I think we can improve spaces here to be consistent with the whole application

Comment (lpelizzaMS) [reply ReactNative/groceryList/src/actions/ProductActions.js:39]: I have changed the problem with the spaces/tabs in the PR 31. I will update this pr when the PR-31 be merged.

File: ReactNative/groceryList/src/components/ProductItem.js (hunk 1 above)
Comment (lpelizzaMS) [reply ReactNative/groceryList/src/components/ProductItem.js:33]: Sure, I just follow the tutorial. Maybe, we can do this kind of improvement in the future. So we will be able to create a new branch with all these improvements. What do you think?

**Pull request merged**